ext/src/dll/make_def.ps1

```diff
@@ -0,0 +1,8 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+param ($dumpbin, $bindir, $inputfile, $targetfile)
+   
+ "LIBRARY opentelemetry_cpp`r`nEXPORTS`r`n" > $targetfile
+    
+ Get-ChildItem -Verbose -Path $bindir/*.lib -Recurse | % { & "$dumpbin" /SYMBOLS $_ | Select-String -Pattern @(Get-Content -Verbose -Path "$inputfile" | Where-Object { $_.Trim() -ne '' } | % { "External\s+\|\s+(\?+[0-9]?$_[^\s]*)\s+\((.*)\)$" }) | % { "; $($_.matches.groups[2])`r`n$($_.matches.groups[1])" } >> $targetfile }
```

Comment:
Can the search path be limited to the subset of directories? I think only `exporters` and `sdk` folders should be included.

Reply:
@ThomsonTan ok, I have pushed changes to accomplish this